samples/bluetooth/peripheral_fast_pair/src/main.c

```diff
@@ -200,13 +230,12 @@ static void volume_control_btn_handle(uint32_t button_state, uint32_t has_change
 
 static void fp_adv_mode_btn_handle(uint32_t button_state, uint32_t has_changed)
 {
-	struct k_work_sync sync;
 	uint32_t button_pressed = button_state & has_changed;
 
 	if (button_pressed & FP_ADV_MODE_BUTTON_MASK) {
-		k_work_cancel_delayable_sync(&fp_adv_mode_status_led_handle, &sync);
+		k_work_cancel_delayable(&fp_adv_mode_status_led_handle);
```

Comment:
This line can be removed. We later reschedule this work anyway and we are in cooperative thread.